src/pages/LendingTeam/Leaderboards.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+	<div class="tw-bg-primary tw-mb-4 tw-rounded">
+		<h3 class="tw-text-h4
+			tw-bg-action
+			tw-text-primary-inverse
+			tw-w-full
+			tw-pl-1
+			tw-rounded-sm"
+		>
+			<template v-if="!isNewMembers">
+				Amount Funded:
+			</template>
+			<template v-else>
+				Member Count:
+			</template>
+		</h3>
+		<kv-tabs @tab-changed="handleTabChanged">
+			<template #tabNav>
+				<kv-tab for-panel="thisMonth"
+					class="tw-text-small"
+				>
+					This Month
+				</kv-tab>
+				<kv-tab for-panel="lastMonth"
+					class="tw-text-small"
+				>
+					Last Month
+				</kv-tab>
+				<kv-tab for-panel="allTime"
+					class="tw-text-small"
+				>
+					All Time
+				</kv-tab>
+			</template>
+			<template #tabPanels>
+				<kv-tab-panel id="thisMonth">
+					<p>Coming Soon...!</p>
```

Comment:
Needs API endpoint to fetch data 
This Month: Amount Funded and New Members 
Last Month: Amount Funded and New Members